vimiv/api/working_directory.py

```diff
@@ -161,14 +164,18 @@ def _on_monitor_fs_changed(self, value: bool) -> None:
     def _load_directory(self, directory: str) -> None:
         """Load supported files for new directory."""
         self._dir = directory
-        self._images, self._directories = self._get_content(directory)
```

Comment:
Did you remove the assignment of `self._directories` by purpose? For me (i.e. without modifying the config or anything), this leads to no directories getting listed in the library.

Reply:
I reproduced your issue- this was most-certainly a careless deletion on my part. Thank you for catching it. Of all the changes, those to 'working_directory.py' will probably need the most careful scrutiny. For whatever reason, I had the most difficult time here.